dffml/util/cli/cmds.py

```diff
@@ -70,31 +72,51 @@ def __init__(self, *args, **kwargs):
                 self.sources[i] = self.sources[i].withconfig(self.extra_config)
 
 
+
+@config
+class ModelCMDConfig(CMDConfig):
+    model: Model = field(
+        "Model used for ML",
+        default_factory = Model.load,
```

Comment:
This shouldn't have a default. It should be required

```suggestion
        required=True,
```

Reply:
Not providing a default argument here throws a `TypeError: non-default argument 'model' follows default argument` because of `ModelCMDConfig` inheriting from `CMDConfig` which has log with a default value.

Reply:
In `dffml.base:config`,

Before calling

https://github.com/intel/dffml/blob/89401c86f959251f05881a5b23336f1c27712cb0/dffml/base.py#L238

We need to make a `dict`, loop through `cls.__dict__.get('__annotations__', {})`, and check if the key exists in `cls.__dict__`. If it does and it's a `field`, then if it doesn't have a `default` add it to the new `dict`. Then loop through again, do the same thing for fields with a default. Now assign the `cls.__annotations = ` the new `dict`.